columnar/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "tantivy-columnar"
```

Comment:
columnar is a little generic name. it's more like dynamic column or json-columnar